charts/prometheus-adapter/templates/_helpers.tpl

```diff
@@ -24,6 +24,13 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Allow the release namespace to be overridden for multi-namespace deployments in combined charts
+*/}}
+{{- define "k8s-prometheus-adapter.namespace" -}}
+{{- default .Release.Namespace .Values.namespaceOverride -}}
```

Comment:
We should add `namespaceOverride: ""` to `values.yaml` and document it.

Reply:
Added